rocksdb_admin/helix_client.cpp

```diff
@@ -47,6 +50,7 @@ JNIEnv* createVM(const std::string& class_path) {
   options[1].optionString = "-verbose:jni";
   // use -Xrs to block JVM signal handling
   options[2].optionString = "-Xrs";
+  options[3].optionString = const_cast<char*>(FLAGS_helix_log_config_file_path.c_str());
```

Comment:
could you please verify that it will work fine even the gflag is set to empty string?

Reply:
Updated the pull request, and tested with empty string and arbitrary string, jvm will ignore the wrong config.